src/sgame/sg_admin.cpp

```diff
@@ -3787,7 +3787,7 @@ bool G_admin_listplayers( gentity_t *ent )
 	char            *registeredname;
 	char            lname[ MAX_NAME_LENGTH ];
 	char            bot, muted, denied;
-	int             authed = 1;
+	int             authed;
```

Comment:
typically one would just remove the declaration here to put it where you initialise the variable. This avoids bugs like the one you fixed

Reply:
I think a bigger refactor to the code could resolve this (as well as move the rest of the variables to suitable locations).

As per IRC discussion, will leave this as-is for now.